### DISCUSSION TITLE
Proof keeps getting rejected at preprocessing stage

### DISCUSSION THREAD

**Discussion by aqumu** [community/community#39495] [GitHub Education]
I tried to register as NSU student and registered appropriate email account. I was asked for additional proof and all of the things I uploaded (i.e. student ID, studying certificate and even a screenshot of a portal, which clearly contains all the information needed) were rejected after preprocessing. It just says that my photo doesn't contain date, my name and organization name. All of the things I tried uploading are attached.
_Edited: Removal of PII_

**Comment by samus-aran** [comment]
Hi @aqumu 👋 

I've removed your private information from [this post](https://github.com/orgs/community/discussions/39495) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. This also applies if the language is not English. Recommendation is to translate the important parts of the evidence in English and show the translation alongside the original items of evidence.

Let us know how you get on 👍
